toolvoice/scripts/inference/qwen_omni_infer.py

```
@@ -0,0 +1,190 @@
+    except (ValueError, SyntaxError):
+        return None
+
+
+def run_row(llm, processor, sampling_params, model_version, benchmark, messages, tools, audio_path):
+    text = processor.apply_chat_template(messages, tools=tools, tokenize=False, add_generation_prompt=True)
+
+    from qwen_omni_utils import process_mm_info
+
+    try:
+        audios, _, _ = process_mm_info(messages, use_audio_in_video=False)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)